ClientSideEncryptionToServerSideEncryptionMigrationSamples/ClientSideKeyVaultKeyToCustomerProvidedKey/README.md

```diff
@@ -1,20 +1,23 @@
 # KeyVault Client Side Encryption Migration to Customer Provided Key Server Side Encryption
 
-This program functions as a migration of data in Azure Storage from client side encryption with KeyVault to server side encryption using a Customer Provided Key for server side encryption.
+This script functions as a sample of how to migrate data that was uploaded using client-side encryption with a local key to use server-side encryption with with a Customer-Provided Key. The sample includes a script to generate sample data, `create_sample_data.py`, as well as a script that will migrate all data within a given container, `miration.py`.
```

Comment:
Nit: You have two occurrences of `with` (probably typo) and also I am assuming it's `migration.py` not miration 😄 

Also how it reads now is fine, but if you were open to suggestions:
"This script functions as a sample of migrating data that was previously uploaded using client-side encryption with a local key to now use server-side encryption with a Customer-Provided Key."

Only reason being is to emphasize that this sample mostly serves people who **previously** had data uploaded, and to hopefully subconsciously discourage new users from digging into these samples. But again, reads fine as is!

Reply:
Edit: Also noticing that this is copy pasted to other READMEs. So, if you do take the feedback on this README, be sure to duplicate your efforts across the other READMEs!